client/app/components/form/MultiStepButtons.tsx

```diff
@@ -28,6 +29,11 @@ const SubmitButton: React.FunctionComponent<SubmitButtonProps> = ({
 }) => {
   const isFinalStep = step === steps.length - 1;
   const isDisabled = disabled || isSubmitting;
+  const { data: session } = useSession();
+
+  const isCasInternal =
+    session?.user.app_role?.includes("cas") &&
+    !session?.user.app_role?.includes("pending");
```

Comment:
I suggest changing this to ` const isIndustryUser = session?.user.app_role?.includes("industry");`

Same with the refactored `UserOperatorForm`, I noticed that this was removed since I added it in a recent PR to enhance the logic to display the new operator message.

The issue with using `!isCasInternal` is that using the client session takes a second or two to load so it will flash for a second if logged in as an internal user and look buggy. I confirmed this by logging in as internal user, scrolling to the bottom of a user operator form and refreshing the page.

If we are specific to who this displays for then we don't have the components flashing for internal users.

Reply:
So using !isCasInternal or !isIndustryUser is going to cause a flash, but isCasInternal or isIndustryUser works? Or using isIndustryUser/!isIndustryUser everywhere works?

Reply:
I think just being specific to who we display the components to will stop components flashing for the wrong users. So avoid using `!isCasInternal` or `!isIndustryUser`. It's something I just recently noticed so there are likely other places to update this, I just noticed it removed with the new `UserOperatorForm` refactor.

Unfortunately there will still be some component pop in as the session loads, though it doesn't look too buggy when it loads in and stays there for the user since we were specific about the users we display it with.

Shon and I recently experimented with passing the server session to the client so there would be no loading issues whatsoever though for now it seems to break CI and e2e tests for some reason so it's a nice to have.

Reply:
Huh, odd. At least we've got a solution even if it's not perfect